data-mgmt-saml-auth/pom.xml

```diff
@@ -48,19 +30,11 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-war-plugin</artifactId>
-                <version>2.4</version>
-                <configuration>
-                    <overlays>
-                        <overlay>
-                            <groupId>com.redhat.lightblue.applications</groupId>
-                            <artifactId>apps-common</artifactId>
-                        </overlay>
-                    </overlays>
-                </configuration>
```

Comment:
Ah, this is overlay of common.  Thought we did an overlay of the unauthenticated app originally.  Nevermind!